test/karma-setup.js

```diff
@@ -448,29 +444,24 @@ function getSVG(chart) {
  * Compares the image data of two canvases
  * @param  {Array} data1 Pixel data for image1.
  * @param  {Array} data2 Pixel data for image2.
- * @return {Number}      The amount of different pixels, where 0 is idenitcal
+ * @return {Number}      The amount of different pixels, where 0 is identical
  */
 function compare(data1, data2) { // eslint-disable-line no-unused-vars
     var i = data1.length,
-        diff = 0,
-        pixels = [],
-        pixel;
+        diff = 0;
 
     // loops over all reds, greens, blues and alphas
     while (i--) {
-        pixel = Math.floor(i / 4);
-        if (Math.abs(data1[i] - data2[i]) !== 0 && !pixels[pixel]) {
-            pixels[pixel] = true;
+        if (!data1[i] === data2[i]) {
```

Comment:
By removing the check for `pixels[pixel]`, we are now no longer counting the amount of different pixels, but the amount of different color values. Each pixel has a red, green, blue and alpha value. So the resulting number will now be up to four times the actual amount of changed pixels. Is that intended?

Reply:
Sorry, that was a mistake on my end. Clearly it is RGBA values. I have reverted the change now.